decompiler/pipeline/controlflowanalysis/loop_utility_methods.py

```diff
@@ -210,3 +228,85 @@ def _requirement_without_reinitialization(ast: AbstractSyntaxTree, node: Abstrac
                 return True
             elif variable in assignment.requirements:
                 return True
+
+
+def _get_equalizable_last_definitions(loop_node: WhileLoopNode, continuation: AstInstruction) -> List[CodeNode]:
+    """
+    Finds equalizable last definitions of the continuation instruction in the code nodes of a while loop containing continue statements.
+
+    :param loop_node: While-loop to search in
+    :param continuation: Instruction defining the for-loops modification
+    :return: List of equalizable last definitions, None if at least one continue node does not match the requirements
+    """
+    equalizable_nodes = []
+    for code_node in (
+        node for node in loop_node.body.get_descendant_code_nodes_interrupting_ancestor_loop() if node.does_end_with_continue
+    ):
+        if not _is_expression_simple_binary_operation(continuation.instruction.value):
+            return None
```

Comment:
Done with [6a9e4bc](https://github.com/fkie-cad/dewolf/pull/326/commits/6a9e4bc2cd2fab14bb21835fb0bf385f15cf9b7f). I also moved the check for the used and defined variable into the simple binary check because they are only used together.